### DISCUSSION TITLE
Folders and child pages are not visible in the nav :(

### DISCUSSION THREAD

**Discussion by dario-l** [retypeapp/retype#497] [General]
Ive just started new project. Added `index.md` in the root. Added folder `guides`. In this folder added another `index.md`. When I run dotnet watch the nav is empty. There is no folder structure visible. :(

I do not know what should I do.

![image](https://user-images.githubusercontent.com/147718/217515763-28724ac0-2d01-4c81-9468-174b5daf5484.png)

UPDATE:
I use dotnet 2.4 version.

Logs:
![image](https://user-images.githubusercontent.com/147718/217553676-440826fa-5ce9-4e4c-9552-c6a954ec1e69.png)

Retype processed 2 pages but only one is visible and available.

**Comment by dario-l** [comment]
Found it. Created md files should not have `page` layout.

I did not found any word in the documentation about this. 😞

**Reply by geoffreymcgill** [reply]
Hi @dario-l. Which layout did you have configured on your `.md` file? I'm assuming the `page` layout?

https://retype.com/configuration/page/#layout

There is a `layout: page` option, although the docs do state `Similar to default layout, but is not added to the main navigation.`. So setting with `layout: page` would cause Retype to not add the page to the left sidebar tree navigation. 

In general, if you are just adding regular documentation pages to your project, setting the `layout` is not required. 

Hope this helps.

**Comment by dario-l** [comment]
@geoffreymcgill thanks 👍🏻
